src/base/CMakeLists.txt

```diff
@@ -168,3 +168,7 @@ if (APPLE)
     find_library(AppKit_LIBRARY AppKit)
     target_link_libraries(qbt_base PRIVATE ${Carbon_LIBRARY} ${IOKit_LIBRARY} ${AppKit_LIBRARY})
 endif (APPLE)
+
+if (WIN32)
+    target_link_libraries(qbt_base PRIVATE advapi32 iphlpapi ole32 powrprof shell32 user32 wsock32 ws2_32)
+endif(WIN32)
```

Comment:
should you add a space before `(`?

Reply:
There is no such space anywhere else in the file or in any other CMakeLists.txt

Reply:
Seriously??
https://github.com/qbittorrent/qBittorrent/blob/e9751e4c5cf59a1883b41c19e899736d7b3a0216/src/base/CMakeLists.txt#L170
`endif` + 'space' + `(APPLE)`

Reply:
XD sorry I thought you were referring `target_link_libraries` 👀😛

Reply:
done.